cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConfiguration.java

```diff
@@ -31,10 +32,14 @@ public class KafkaConfiguration extends AbstractConfiguration {
 
     private static final List<String> FORBIDDEN_PREFIXES;
     private static final List<String> FORBIDDEN_PREFIX_EXCEPTIONS;
+    private static final Map<String, String> DEFAULTS;
 
     static {
         FORBIDDEN_PREFIXES = AbstractConfiguration.splitPrefixesOrOptionsToList(KafkaClusterSpec.FORBIDDEN_PREFIXES);
         FORBIDDEN_PREFIX_EXCEPTIONS = AbstractConfiguration.splitPrefixesOrOptionsToList(KafkaClusterSpec.FORBIDDEN_PREFIX_EXCEPTIONS);
+
+        DEFAULTS = new HashMap<>(6);
```

Comment:
```suggestion
        DEFAULTS = new HashMap<>();
```

(Or `1` if you want, I guess ...)